streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -2773,10 +2894,8 @@ public void punctuateShouldThrowStreamsExceptionWhenProcessingExceptionHandlerRe
         when(stateManager.taskId()).thenReturn(taskId);
         when(stateManager.taskType()).thenReturn(TaskType.ACTIVE);
         task = createStatelessTask(createConfig(
-            AT_LEAST_ONCE,
             "100",
-            LogAndFailExceptionHandler.class.getName(),
```

Comment:
Why remove here?

Reply:
Just simplifying the setup: added a new `createConfig` overload which does set `LogAndFailExceptionHandler` by default (instead of using the old one which always get `LogAndFailExceptionHandler` passed when used)